tiatoolbox/models/architecture/nuclick.py

```diff
@@ -285,7 +288,8 @@ def __init__(
         strides: Union[Tuple[int, int], np.ndarray] = (1, 1),
         use_bias: bool = False,
         dilation_rate: Union[Tuple[int, int], np.ndarray] = (1, 1),
-    ):
+    ) -> None:
+        """Initializes :class:`ResidualConv`."""
```

Comment:
```suggestion
        """Initialize :class:`ResidualConv`."""
```

Reply:
Addressed in this commit https://github.com/TissueImageAnalytics/tiatoolbox/pull/625/commits/c2e1cd4ccfd6aaeb32664f23d4f6634cddb90090